stable/minio/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: MinIO is a high performance distributed object storage server, designed for large-scale private cloud infrastructure.
 name: minio
-version: 2.5.2
-appVersion: RELEASE.2019-07-17T22-54-12Z
```

Comment:
lets keep the `appVersion`  as RELEASE.2019-07-17T22-54-12Z

Reply:
My bad - fixed.